engine-cockpit/webContent/resources/composite/SslTable.xhtml

```diff
@@ -26,12 +26,17 @@
           </p:commandButton>
         </div>
       </div>
+      <p:staticMessage
+        rendered="#{not empty cc.attrs.bean.certificateLoadError}"
+        id="certificateLoadError" severity="warn" 
+        detail="#{cc.attrs.bean.certificateLoadError}" />
```

Comment:
I think I would show here a growl and not a static message...
And I would make the exception message the detail of the growl and the title something which is translated, e.g. "Error while accessing the keystore"

Reply:
No wenn an exception raises then show a growl instead of this static message.

Reply:
or better yes maybe. when the save also can have exceptions then yes. but also on all other cases.
Just like you do it now, but instead of set this exception message and hold it in the bean, just send a message via the MessageUtil. And maybe you need to add a p:growl component to the view